### ISSUE TITLE
Sound crackles in Yosemite

### ISSUE THREAD

**Issue by sneslink** [nevyn/cfxr#5]
In case anyone wants to continue working on CFXR: I have frequent crackles and pops when previewing sounds. The noise is not present in exported sounds, but it makes shaping sound effects very difficult.

**Comment by alelepd** [comment]
Yes, many lines of code are now deprecated is latest OSX versions :/
It needs debugging but I think it something on the playback.m file.
